_faq/application_template.md

```diff
@@ -23,13 +23,16 @@ Student Info
 Code Sample
 -----------
 
-Link all your contributions to coala and any other open source project here.
+- Link all your contributions to coala here.
+
 For example, to list your contributions to coala you can mention:
+  - https://github.com/coala/coala/commits/master?author=YOUR_USER_NAME
+    for coala repo
+  - https://github.com/coala/coala-bears/commits/master?author=YOUR_USER_NAME
+    for coala-bears repo
```

Comment:
In that case, should I remove indentation here?

Reply:
No, I said "and then I think it is good enough for now."  that means the rest is fine.